css/styles.css

```diff
@@ -0,0 +1,84 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo a,
+.menu a {
+  font-weight: 700;
+}
+
+.menu {
+  display: flex;
+  justify-content: end;
+}
+
+.flex-nav {
+  display: flex;
+  flex-flow: row wrap;
+  list-style: none;
+  justify-content: space-between;
+  padding: 10px;
+  margin: 1.8rem;
+}
+
+.bg_shape {
+  position: absolute;
+  z-index: -1;
+  width: 100%;
+  margin-top: 6.25rem;
+}
+
+body {
+  font-family: sans-serif;
+  margin: 0;
+}
+
+.social {
+  margin: 1rem 0;
+  height: 20px;
+  display: flex;
+  padding: 0;
+  margin-top: 1rem;
+  align-items: baseline;
+}
+
+.social a {
+  margin-right: 1rem;
+}
+
+.social img {
+  width: 1.5rem;
+}
+
+.social li {
+  list-style: none;
+}
+
+.wrap {
+  margin: 2rem;
+}
```

Comment:
- Kindly add some paddings to your section so it matches the design as much as possible.

Reply:
@zieeco Update done